src/integration-test/java/org/ow2/proactive/catalog/rest/controller/CatalogObjectControllerIntegrationTest.java

```diff
@@ -168,729 +168,753 @@ public void testCreateWorkflowShouldReturnSavedWorkflow() {
-        given().pathParam("bucketName", bucket.getName())
-               .pathParam("name", "workflow_new")
-               .when()
-               .get(CATALOG_OBJECT_RESOURCE)
-               .then()
-               .assertThat()
-               .statusCode(HttpStatus.SC_OK)
-               .body("commit_message", is(archiveCommitMessage))
-               .body("content_type", is(MediaType.APPLICATION_XML.toString()))
-               .body("extension", is("xml"));
-    }
-
-    @Test
-    public void testCreateObjectDifferentTypeFromArchive() {
-        String archiveCommitMessage = "Import from archive";
-
-        //Create objects from the archive
-        given().header("sessionID", "12345")
-               .pathParam("bucketName", bucket.getName())
-               .queryParam("kind", "my-kind")
-               .queryParam("commitMessage", archiveCommitMessage)
-               .queryParam("objectContentType", MediaType.MULTIPART_FORM_DATA.toString())
-               .multiPart(IntegrationTestUtil.getArchiveFile("filesGCP.zip"))
-               .when()
-               .post(CATALOG_OBJECTS_RESOURCE)
-               .then()
-               .assertThat()
-               .statusCode(HttpStatus.SC_CREATED)
-               .body("object", hasSize(12));
-
-        //Check that the object was created
-        given().pathParam("bucketName", bucket.getName())
-               .pathParam("name", "cmdFile")
-               .when()
-               .get(CATALOG_OBJECT_RESOURCE)
-               .then()
-               .assertThat()
-               .statusCode(HttpStatus.SC_OK)
-               .body("commit_message", is(archiveCommitMessage))
-               .body("content_type", is("application/x-bat"))
-               .body("extension", is("bat"));
-
-        //Check that the object was created
-        given().pathParam("bucketName", bucket.getName())
-               .pathParam("name", "array")
-               .when()
-               .get(CATALOG_OBJECT_RESOURCE)
-               .then()
-               .assertThat()
-               .statusCode(HttpStatus.SC_OK)
-               .body("commit_message", is(archiveCommitMessage))
-               .body("content_type", is(MediaType.APPLICATION_JSON_VALUE))
-               .body("extension", is("json"));
-    }
-
-    @Test
-    public void testCreateWorkflowsFromArchiveWithBadArchive() {
-        given().header("sessionID", "12345")
-               .pathParam("bucketName", bucket.getName())
-               .queryParam("kind", "workflow")
-               .queryParam("commitMessage", "Import from archive")
-               .queryParam("objectContentType", MediaType.APPLICATION_XML.toString())
-               .multiPart(IntegrationTestUtil.getArchiveFile("workflow_0.xml"))
-               .when()
-               .post(CATALOG_OBJECTS_RESOURCE)
-               .then()
-               .assertThat()
-               .statusCode(HttpStatus.SC_UNPROCESSABLE_ENTITY);
-    }
+    /*
```

Comment:
Commenting tests makes no sense, you can use @Ignore with an explanation.

At any rate, why these tests must be removed ? We reduce the testing span doing this, and our ability to spot regressions.

Reply:
I commented tests and forgot about them. Block comment removed and tests work fine. Sorry for that.